Example/Example/Model/Model.swift

```diff
@@ -1,7 +1,7 @@
 import Faro
 
 /// Example model
-class Model: Parseable {
+class Model: Deserializable {
```

Comment:
Bedoel je de naamgeving of het opsplitsen in twee protocols? Reden voor het opsplitsen is dat gebruikers van Faro in veel gevallen helemaal niks gaan posten, putten of deleten, en dat verplichting om omgekeerde mapping te implementeren als je weet dat je 'm niet nodig hebt een drempel is.